company-policies/sustainability.md

```diff
@@ -13,11 +13,11 @@ Sustainability has been one of the consistent threads in our work over the last
 
 ### We believe
 
-- Our ecosystem is interdependent
-- Using science, we can understand environmental systems and effectively address issues
-- Promoting habitat and diversity, supports ecological systems
-- Environmental challenges have social and political impacts
-- The impact of climate change will disproportionately affect disadvantaged groups
+-   Our ecosystem is interdependent
```

Comment:
@mgifford sorry about this. Looks like we need to adjust configuration for our remark linter https://github.com/remarkjs/remark-lint/blob/main/packages/remark-lint-list-item-indent/readme.md#recommendation

Reply:
@grugnog do you know what might be doing this? pre-commit keeps switching the spacing from 1 space to 3 total spaces.

Reply:
Ah it is pre-commit and prettier that is messing with this.

Reply:
I think updating remark-lint-list-item-indent will fix this but I need to test it some more.

Reply:
This is resolved and I also put the remark updates in #1388 so it can be merged separately